pagopa/io-app
# [#172077326] Mapping FAQ Categories on related screens

**Short description:**
This PR adds faq categories to the related screen.

**How to test:**
Click on the question mark button on the Header of any screen to check if the faqs display on the contextual help component
